tasks/msi.py

```diff
@@ -272,13 +273,15 @@ def _build_msi(ctx, env, outdir, name, allowlist):
 
 def _msi_output_name(env):
     if _is_fips_mode(env):
-        return f"datadog-fips-agent-{env['PACKAGE_VERSION']}-1-x86_64"
+        return f"datadog-fips-agent-{env['UPGRADE_TEST']}{env['PACKAGE_VERSION']}-1-x86_64"
     else:
-        return f"datadog-agent-{env['PACKAGE_VERSION']}-1-x86_64"
+        return f"datadog-agent-{env['UPGRADE_TEST']}{env['PACKAGE_VERSION']}-1-x86_64"
```

Comment:
suggestion: use a more descriptive env var name. e.g. `AGENT_PRODUCT_NAME_SUFFIX`, would also be usable for other things